website/source/layouts/guides.erb

```diff
@@ -65,6 +65,9 @@
           <li<%= sidebar_current("guides-identity-lease") %>>
             <a href="/guides/identity/lease.html">Tokens and Leases</a>
           </li>
+          <li<%= sidebar_current("guides-identity-identity") %>>
+            <a href="/guides/identity/identity.html">Identity - Entities & Groups</a>
```

Comment:
Just to be safe (and since we do above in this file), can we use `&amp;` here instead of `&`?

Reply:
Nice catch.  Thank you!!!!  It's fixed.